tests/test_preferences.py

```diff
@@ -0,0 +1,15 @@
+import ansys.fluent.core as pyfluent
+
+
+def test_solver_verbosity():
```

Comment:
@hpohekar have a look at other tests.  You will need to use a _fixture_ for the launch part. Have a look at test_creatable.py for instance.

Reply:
Done. Thank you.

Reply:
![image](https://user-images.githubusercontent.com/106588300/189635823-9c4dcdc8-c93d-4f6b-a6a9-8909dc7b4483.png)